etl-routes.js

```diff
@@ -189,6 +189,36 @@ module.exports = function () {
   },
   {
     method: 'GET',
+    path: '/etl/clinical-reminder/{referenceDate}',
```

Comment:
This needs to be done by patient i.e
/etl/patient/{patient-uuid}/hiv-clinical-reminders/{referenceDate}